opal/mca/btl/base/btl_base_am_rdma.c

```diff
@@ -445,39 +443,35 @@ mca_btl_base_rdma_start (mca_btl_base_module_t *btl, struct mca_btl_base_endpoin
     context->local_handle   = local_handle;
     context->total_size     = size;
 
-    size_t send_size = 0;
-    size_t recv_size = 0;
```

Comment:
@hjelmn This was marked as unused, was this intended to get added to the packet_size or should this and this block be removed:
```
else if (MCA_BTL_BASE_AM_GET == type) {
        if (sizeof (mca_btl_base_rdma_response_hdr_t) + size <= btl->btl_eager_limit) {
            recv_size = size;
            packet_size += size;
        } else if (!mca_btl_base_rdma_use_rdma_put (btl)) {
            recv_size = size_t_min (size, btl->btl_max_send_size -
            packet_size += size_t_min (size, btl->btl_max_send_size -
                                              sizeof (mca_btl_base_rdma_response_hdr_t));
        } else {
            use_rdma = true;
        }
    } else {
        /* fetching atomic. always recv result via active message */
        recv_size = size;
        packet_size += size;
    }
```

Reply:
@hjelmn when you get a chance can you look at the above?